release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2892,6 +2892,42 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.30
+[id="ocp-4-17-30_{context}"]
+=== RHSA-2025:7669 - {product-title} {product-version}.30 bug fix and security update advisory
+
+Issued: 21 May 2025
+
+{product-title} release {product-version}.30 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:7669[RHSA-2025:7669] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:7671[RHBA-2025:7671] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.30 --pullspecs
+----
+
+[id="ocp-4-17-30-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, no event was logged when an error occurred from a failed ingress-to-route conversion. With this update, this error appears in the `event` logs. (link:https://issues.redhat.com/browse/OCPBUGS-55943[*OCPBUGS-55943*])
```

Comment:
```suggestion
* Previously, no event was logged when an error occurred from a failed ingress-to-route conversion. With this update, this error appears in the `event` logs. (link:https://issues.redhat.com/browse/OCPBUGS-55943[OCPBUGS-55943])
```
I think we're no longer highlighting Jiras in bold. If I'm wrong, please ignore this suggestion and my other similar suggestions :)